scalardb-analytics-spark-sample/README.md

```diff
@@ -0,0 +1,5 @@
+# Sample Application of ScalarDB Analytics with Spark
+
+This sample application demonstrates how you can run analytical queries on data that is managed by [ScalarDB](https://www.scalar-labs.com/scalardb/) by using [ScalarDB Analytics with Spark](https://scalardb.scalar-labs.com/docs/latest/scalardb-analytics-spark/getting-started/).
+
+For more information, please refer to the [tutorial](https://scalardb.scalar-labs.com/docs/latest/scalardb-samples/scalardb-analytics-spark-sample/).
```

Comment:
It looks like this document is not found. Do you plan to add it in another PR in the internal docs repo?